webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -94,10 +94,13 @@ def timestamp_to_date(
 def time_tag(dt, format='%a, %b %d %H:%M %Z', future=False):
     if not isinstance(dt, (datetime.date, datetime.datetime)):
         try:
+            if isinstance(dt, unicode):
+                # isodate struggles to convert unicode strings with
+                # its parse_datetime() if the input string is unicode.
+                dt = dt.encode('utf-8')
```

Comment:
done.

On Wed, Sep 7, 2016 at 12:23 PM, Will Kahn-Greene <notifications@github.com>
wrote:

> In webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py
> <https://github.com/mozilla/socorro/pull/3457#discussion_r77856629>:
>
> > @@ -94,10 +94,13 @@ def timestamp_to_date(
> >  def time_tag(dt, format='%a, %b %d %H:%M %Z', future=False):
> >      if not isinstance(dt, (datetime.date, datetime.datetime)):
> >          try:
> > +            if isinstance(dt, unicode):
> > +                # isodate struggles to convert unicode strings with
> > +                # its parse_datetime() if the input string is unicode.
> > +                dt = dt.encode('utf-8')
>
> Mmm... I think there aren't any cases where there are valid unicode
> characters in an iso-standard date format. I think we should do
> .encode('ascii'). That will likely throw an encoding error for bad data,
> but only for bad data that is likely not a timestamp.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mozilla/socorro/pull/3457/files/fc7b6a12a6bcea5f2ed6da415348f11cef9ad995#r77856629>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABoc1Iuts4gpIAaGky8lxAnMsVy0eA3ks5qnuTvgaJpZM4J3GwG>
> .
>



-- 
Peter Bengtsson
Mozilla Tools & Services